src/org/traccar/api/resource/NotificationResource.java

```diff
@@ -49,7 +49,7 @@
             userId = getUserId();
         }
         Context.getPermissionsManager().checkUser(getUserId(), userId);
-        return Context.getNotificationManager().getUserNotifications(userId);
+        return Context.getNotificationManager().getAllUserNotifications(userId);
```

Comment:
We can't remove it. I remember, we use it to generate list of event types for `ReportConfig` dialog